starlite/events/listener.py

```diff
@@ -0,0 +1,38 @@
+from typing import TYPE_CHECKING, Any, List, Union
+
+from starlite.exceptions import ImproperlyConfiguredException
+from starlite.utils import AsyncCallable
+
+if TYPE_CHECKING:
+    from starlite.types import AnyCallable
+
+
+class EventListener:
+    """Decorator for event listeners"""
+
+    __slots__ = ("event_ids", "fn", "listener_id")
+
+    fn: "AsyncCallable[Any, Any]"
+
+    def __init__(self, event_id: Union[str, List[str]]):
```

Comment:
Why not just 

```suggestion
    def __init__(self, *event_ids: str):
```

?